generator/internal/genclient/language/internal/rust/rust.go

```diff
@@ -86,11 +86,17 @@ func NewCodec(copts *genclient.CodecOptions) (*Codec, error) {
 				specificationPackages = append(specificationPackages, s[1])
 			case "feature":
 				pkg.Features = append(pkg.Features, strings.Split(s[1], ",")...)
+			case "ignore":
+				value, err := strconv.ParseBool(s[1])
+				if err != nil {
+					return nil, fmt.Errorf("cannot convert `ignore` value in %s to boolean: %w", definition, err)
```

Comment:
nit: in other error message you would have written `value (%s) to boolean`. Also maybe the parenthesis are fine but you could also do `%q` for a quoted string

Reply:
Done.